src/content/docs/errors-inbox/apm-tab.mdx

```diff
@@ -0,0 +1,103 @@
+* **Profiles:** The profiles tab helps you observe interesting trends across all error occurrences inside a specific error group. You can download this information as a CSV file for further analysis. When you click on an attribute bar, a table is rendered showing the breakdown of attributes and counts.
+
+<img
+  title="error profiles"
+  alt="A depiction of error profiles in the UI"
+  src={apmErrorProfiles}
+/>
+
+### Error occurrences: [#error-occurrences]
+
+The detailed view also allows you to view specific errors. In the top right, you can navigate between the first instance of the error, the last, and any instance in between.
+
+## Triage [#triage-card]
+
+The triage section links the specific error occurrence you're viewing to a system-created error group that has a unique fingerprint. Why does that matter? That unique fingerprint  enables you to triage an error group using a status update or assignment. The system-created error groups are the ones you find on the Triage tab. For more info on how they are generated, see How error groups work.
+
+<img
+  title="triage errors"
+  alt="A depiction of how to tirage errors in APM"
+  src={apmTriageErrors}
+/>
+
+## Select the time period for error data [#selected-time-period]
+
+Use the time picker to examine the details of error events over the past week, month, etc. 
+
+* Error event data in the **Group errors tab** is available for up to a 7 day window of data collected over the last eight days. 
+* Error event metadata in the **Triage tab** is stored up to 13 months. 
+
+You may notice slight differences in count if your time window is set to ending now. This occurs because the counts for the list and table may be requested at slightly different times as the page auto-refreshes.	
+
+## Expected errors [#expected-errors]
+
+
+Reduce noisy errors by marking them as “expected errors”. Such errors won't affect reported error rate or Apdex metrics. See [Manage errors](/docs/agents/manage-apm-agents/agent-data/manage-errors-apm-collect-ignore-or-mark-expected#expected) for more details.
+
+Expected errors aren't shown by default on the Group errors tab. You can show them by turning on the **Show N expected errors** switch below the **Group by** bar.
+
+
+## Errors outside transactions [#outside-transactions]
+
+Using the New Relic agent API, you can [record custom errors](/docs/agents/manage-apm-agents/agent-data/manage-errors-apm-collect-ignore-or-mark-expected#error-collection) at any point in the execution of your code. Sometimes, such custom errors will occur when a transaction is not executing, such as in high-volume asynchronous code that doesn't handle an HTTP transaction.
+
+Errors outside transactions are shown by default on the Group errors tab, but if you want to view only errors outside of the transaction, apply the following filter: 
```

Comment:
either 'group errors tab' or '**Group errors** tab'. maybe first one is better for general purpose.

Reply:
done